ibis/backends/duckdb/__init__.py

```diff
@@ -125,6 +125,7 @@ def configure_connection(dbapi_connection, connection_record):
             # the progress bar causes kernel crashes in jupyterlab ¯\_(ツ)_/¯
             dbapi_connection.execute("SET enable_progress_bar = false")
 
+        self._record_batch_reader_states = {}
```

Comment:
Can we call this something that indicates what the values represent? Maybe `_readers_consumed`?